source/helpers/hotfix.ts

```diff
@@ -15,10 +17,16 @@ export const updateHotfixes = cache.function(async (): Promise<string[][]> => {
 		return [];
 	}
 
-	return atob(content)
-		.trim()
-		.split('\n')
-		.map(line => line.split(','));
+	const storage: HotfixStorage = [];
+	const lines = atob(content).trim().split('\n');
+	for (const line of lines) {
+		const [featureID, unaffectedVersion, relatedIssue] = line.split(',');
+		if (!unaffectedVersion || compareVersions(unaffectedVersion, version) > 0) {
```

Comment:
We could also check that `featureID` and `relatedIssue` are not empty of `undefined` (that would avoid storing the example line).